dibot-labs/avalon
# Initializes webpack, eslint, package.json and client file structure

#### What's this PR do?

Initialize webpack, eslint, babel, package.json, and client file structure.
#### What are the important parts of the code?

None.
#### How should this be tested by the reviewer?

`webpack -d` to create bundle, then open client/dist/index.html in the browser. Check that 'Hi Avalon!' is displayed and there are no errors in the console. 
#### Is any other information necessary to understand this?

Nope.
#### What are the relevant tickets? (Please add `closes`, `refs`, etc)

None. 
#### Screenshots (if appropriate)
